README.md

```diff
@@ -1,72 +1,38 @@
-
-If you are building on Windows or OSX, building is possible via Docker. (https://hub.docker.com/r/microsoft/dotnet/)
-
-### Build on Windows
-
-```console
-./build.ps1
-```
+> The source-build repository doesn't currently support Windows. See [source-build#1190](https://github.com/dotnet/source-build/issues/1190).
 
 ### Build on Linux or macOS
 
 ```console
 ./build.sh
 ```
 
-##  Script Users
-
-The most common users are expected to be:
-
-* .NET Core contributors.
-* Linux distribution maintainers.
-* Cloud service developers.
-
-You do not have to build the entire product to contribute to .NET Core. Often, you only need to build a single binary to test a change. There are some scenarios where building the whole product is useful, such as adding and testing a feature that requires changes to multiple repos.
+## Goals
 
-## What the Scripts Do
+The key goal of this repository is to satisfy the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html). Many Linux distributions have similar rules. These rules tend to have two main principles: consistent reproducibility, and source code for everything.
 
-The scripts can be thought of as solving challenges that would otherwise making building the whole product difficult. The following challenges are the primary ones that developers often hit before these scripts were available.
+A secondary goal of source-build is to allow .NET Core contributors to build a .NET Core SDK with coordinated changes in multiple repositories. However, the developer experience is significantly better in individual repositories and, if possible, contributors should make and test changes in the target repo, not source-build.
 
-* .NET Core is composed of several repositories that all need to be built.
-* The .NET Core SDK generated by the build requires a specific layout in order to correctly function.
-* Most of the product is written in managed code and requires the .NET Core SDK to build. This approach is a great use of the product, but presents a boot-strapping problem for the build.
+## What does the source-build infrastructure do?
 
-## Goals
+Source-build solves common challenges that most developers encounter when trying to build the whole .NET Core SDK from source.
 
-Many Linux distributions have specific rules for official packages. The rules can be summarized as two main rules: source for everything, and consistent reproducibility.
+* .NET Core is composed of many repositories that need to be built at a specific combination of commits.
+* Each repository's build output needs to flow into the next repository's build.
+* Nearly all .NET Core repositories require the .NET Core SDK to build. This is a circular dependency, which presents a bootstrapping problem.
 
```

Comment:
I would add one more item:

- building repository generally tries to download prebuilt binaries during build